docs/tutorial/image_cls.md

```diff
@@ -106,15 +106,18 @@ If configuration finishes, the configuration file is generated in the `cifar10_t
 Train your model by running `blueoil train` with model configuration.
 
     $ docker run --rm \
-        -e CUDA_VISIBLE_DEVICES=0 \
+        --gpus device=0 \
         -v $(pwd)/cifar:/home/blueoil/cifar \
         -v $(pwd)/config:/home/blueoil/config \
         -v $(pwd)/saved:/home/blueoil/saved \
         blueoil_$(id -un):{TAG} \
         blueoil train -c config/cifar10_test.py
 
 Just like init, set the value of `{TAG}` to the value obtained by `docker images`.
-Change the value of `CUDA_VISIBLE_DEVICES` according to your environment.
+Change the value of `--gpu device=<gpu indices>` according to your environment. 
+For detail check the official [documentation](https://docs.docker.com/config/containers/resource_constraints/#gpu).
+> For old docker versions use `--runtime=nvidia` with `-e CUDA_VISIBLE_DEVICES=<gpu indices>` environment variable
+> instead of `--gpu device=<gpu indices>`.
```

Comment:
Thanks, but I think we don't need these lines because our supported docker version is `Docker >= 19.03`.
https://docs.blue-oil.org/install/install.html#prerequisites

Reply:
OK. I'll remove it. Thanks for the review.